Sources/Orbit/Support/Components/TimelineItem.swift

```diff
@@ -0,0 +1,413 @@
+import SwiftUI
+
+public extension VerticalAlignment {
+
+    enum TimelineItemAlignment : AlignmentID {
+        public static func defaultValue(in dimensions: ViewDimensions) -> CGFloat {
+            return dimensions[VerticalAlignment.top]
+        }
+
+        public static func computeFirstLineCenter(in dimensions: ViewDimensions) -> CGFloat {
+            (dimensions.height - (dimensions[.lastTextBaseline] - dimensions[.firstTextBaseline])) / 2
+        }
+    }
+
+    static let timelineItemAlignment = VerticalAlignment(TimelineItemAlignment.self)
+}
+
+/// One item of a Timeline.
+///
+/// - Related components
+///   - ``Timeline``
+///
+/// - Note: [Orbit definition](https://orbit.kiwi/components/progress-indicators/timeline/)
+public struct TimelineItem<Footer: View>: View {
+
+    @Environment(\.sizeCategory) var sizeCategory
+    @Environment(\.horizontalSizeClass) var horisontalSizeClass
+    @Environment(\.accessibilityReduceMotion) var reduceMotion
+
+    let label: String
+    let sublabel: String
+    let description: String
+    let style: TimelineItemStyle
+    @ViewBuilder let footer: Footer
+
+    var hasHeaderContent: Bool {
+        label.isEmpty == false || sublabel.isEmpty == false
+    }
+
+    var hasDescription: Bool {
+        description.isEmpty == false
+    }
+
+    var animation: Animation {
+        Animation.easeInOut.repeatForever().speed(0.25)
+    }
+
+    @State var animationLoopTrigger = false
+
+    public var body: some View {
+        HStack(alignment: .timelineItemAlignment, spacing: .small) {
```

Comment:
We should reuse the `firstTextBaseline` alignment guide that we use for these. We can just compute the firstTextBaseline for the indicator.

Reply:
Good point, removed